docker/config/test.env

```diff
@@ -6,8 +6,8 @@ alembic_config=/app/docker/config/alembic.ini
 # put filesystem crashstorage stuff in /tmp/test_crashes
 resource.fs.fs_root=/tmp/test_crashes
 
-# use socorro_integration_test for postgres and alembic
-database_url=postgres://postgres:aPassword@postgresql:5432/socorro_integration_test
+# use socorro_test for postgres and alembic
+database_url=postgres://postgres:aPassword@postgresql:5432/socorro_test
```

Comment:
While working on things, I cleaned up db creation, so now we only create 2 databases instead of 3 in the tests. In this way, this PR is both 2-to-3 and also 3-to-2!!!!!